### DISCUSSION TITLE
azure api

### DISCUSSION THREAD

**Discussion by blagoda** [AzureCommunityPL/group#141] [Pomoc]
Witam Mamy pytanko moze ktoś zna przyczynę. Zarejestrowałem użytkownika na google apps, dostłem id loginy itp. Dałem temu użytkownikowi dostep do dysku współdzielonego z uprawnieniami "autor". Następnie korzystajac z azure api prubuję podłączyć sie do tego zasobu wybierając np. wyświetl zawartość katalogów i tu dostaje komunikat  "Nie udało nam się potwierdzić, że to konto jest Twoje." Jakieś pomysły ?

**Comment by kamil-mrzyglod** [comment]
Trochę mało konkretów:

- o jaki dysk Ci chodzi? 
- za pomocą którego Azure API **próbujesz** się podłączyć?
- w jaki sposób wywołujesz to API?
- w jaki sposób nadałeś dostęp do dysku?

**Comment by blagoda** [comment]
chodzi o zwykły dysku współdzielony 
za pomocą api google 
api wywołuje z Logic app disigner 
a urawnienia dodawałem zwyczajnie jak zwkłemu użytkownikowi czyli prawym na dysk wspódzielony potem Manage members dodoaje konto serwisowe i ustawiam mu "Manager" dle tego dysku. 
PS. Nie da sie tutj dodawać zdjęć byłoby jaśniej

**Comment by eklime** [comment]
Załączony obrazek:
![image](https://user-images.githubusercontent.com/12485979/99070312-42325a00-25b0-11eb-8d7f-51dc7ca30dc0.png)

**Reply by eklime** [reply]
Aaa i nieskromnie chciabym zauważyć, że piszesz to o jakimś "google api", "google dysk" itp. i dostajesz komunikaz Z "google" to ja nie wiem czy problem nie jest raczej po stronie "google", a nie Azure ;)
